changelog/may2023/2023-05-12-functions-added-functions-triggers-in-beta.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Functions Triggers in Beta
+status: added
+author:
+    fullname: 'Join the #[serverless-functions] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-05-12
+category: serverless
+product: functions
+---
+
+Functions can now be triggered from Message Queues hosted in Messaging and Queuing SQS namespace.
+Simply add a [SQS Trigger](https://www.scaleway.com/en/docs/serverless/functions/how-to/add-trigger-to-a-function/) from the console and when a new message will be sent to the queue, it will trigger your function.
```

Comment:
```suggestion
Simply add an [SQS Trigger](https://www.scaleway.com/en/docs/serverless/functions/how-to/add-trigger-to-a-function/) from the console and when a new message is sent to the queue, it will trigger your function.
```